website/index.html

```diff
@@ -4,118 +4,130 @@
+    <!--       Header         -->
+    <section class="header">
+        <header>   
+            <svg class="logo"><use xlink:href="#logo_vector"></use></svg>
+            <div class="nav">
+                <a class="nav_link nav_link_stays" href="#">Stays</a>
+                <a class="nav_link nav_link_attractions" href="#">Attractions</a>
+                <a class="nav_link" href="#"><svg class="night"><use xlink:href="#Night"></use></svg></a>
+                <a class="nav_link" href="#"><svg class="account"><use xlink:href="#AccountCircle"></use></svg></a>
+                <a class="nav_link" href="#"><svg class="menu"><use xlink:href="#menu"></use></svg></a>
+            </div>    
+        </header>
+        <div class="intro">
+            <h1>Discover stays <br> to live, work or just relax</h1>
+            <div class="intro_text">
+                <p>Your destination or hotel name</p>
+                <p>Check-in — Check-out</p>
             </div>
         </div>
-
-        <!--       Block      -->
-        <div class="block">
-        <div class="first_block">
-            <h2>What do we offer</h2>
-            <div class="line"></div>
-        </div> 
-        <div class="item_container">
-            <div class="item">
-                <div class="item_wrapper">
-                    <img class="item_icon" src="img/Support.png">
-                </div>
-                <div class="item_title">Support 24/7</div> 
-            </div>
-            <div class="item">
-                <div class="item_wrapper">
-                    <img class="item_icon" src="img/Communicate.png">
-                </div>
-                <div class="item_title">Communicate directly</div> 
-            </div>
-            <div class="item">
-                <div class="item_wrapper">
-                    <img class="item_icon" src="img/Book.png">
-                </div>
-                <div class="item_title">Book online</div> 
+        <form class="search_panel" action="#">
+            <input class="search_input search_line_city" type="text" placeholder="New York">
+        <div class="search_input search_line_date">
+            <input class="search_line_right " type="date" placeholder="Tue 15 Sept-Sat 19 Sept">
+            <input class="search_line_left" type="date" placeholder="Tue 15 Sept-Sat 19 Sept">
+        </div>
+            <input class="search_input search_line_room" type="text" placeholder="2 Adults-0 Children-1Room">
+            <input class="submit" type="submit" value="Search">
+        </form>
+        <div class="downloads">
+            <img class="downloads_content" src="img/app-store-badge.svg" alt="#">
+            <img class="downloads_content" src="img/google-play-badge.svg" alt="#">
+        </div>
+    </section>
+   
+    <!--       Offer      -->
+    <section class="offer_section">
+        <h2 class="offer_title">What do we offer</h2>
+        <div class="line"></div>
+        <div class="offer_container">
```

Comment:
Add top margin for elements
![image](https://user-images.githubusercontent.com/20189313/129492999-9f35e5db-4899-4088-a072-e6447e662d3b.png)